stand-alone/standalone-loader.tsx

```diff
@@ -80,6 +80,7 @@ window.loadPlayer = async (context: RedVenturePlayerContextProps) => {
 			jwPlayerContainerEmbedId: getJwPlayerContainerEmbedId(context),
 			playerUrl: context?.playerUrl,
 			autoPlay: context?.autoPlay,
+			hasAds: context.hasAds,
 		} as RedVentureVideoPlayerProps),
```

Comment:
It seems that the conversion of type `{ videoDetails: RedVentureVideoDetails, ... }` to type `RedVentureVideoPlayerProps` could be a mistake, as these types are not compatible. I suspect that this is intentional. However, should we convert the expression to `unknown` first?

```ts
videoDetails: redVentureVideoDetails as unknown,
```

Currently, it should throw the TS2352 error 🤔

Reply:
As that issue was already there I'd not bother as it's not critical (no runtime issue just types) and we are most likely sunsetting jwplayer-fandom soon :)

Reply:
☝️ that is actually not true -adding `hasAds?: boolean;` to `stand-alone/types.d.ts` resolves the issue. I will add it

Reply:
The error TS2352 has been resolved. Thank you very much!